addons/web_editor/views/snippets.xml

```diff
@@ -464,74 +464,146 @@
                     using xpath to add non-animated shapes, also use xpath to
                     remove the t-if on the page. This is why individual options
                     also have that t-if.  -->
```

Comment:
You need to consider comments, here it's left unrelevant.
Remove the comment + triple check if there is no xpath t-if (you removed it in mass_mailing so should be ok)